frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -1,8 +1,12 @@
 import { SubmissionState } from './types';
 import { EXERCISE_SUBMISSIONS } from './exercise-submissions.actions';
+import getMockSubmissions from '../../services/Helpers';
+import MOCK_SUBMISSIONS from '../../pages/api/exercise-submissions/mock';
 
 const initialState: SubmissionState = {
-  submissions: [],
+  submissions: MOCK_SUBMISSIONS,
```

Comment:
We should add another field here keeping track of the search query. Let's name it `search`. 

Also, we can rename `currentPage` to `page`, I think it's obvious what's going on.